lib_dock.ks

```diff
@@ -127,7 +127,9 @@ function dockChooseDeparturePort {
 // Find suitable docking ports on self and target. Works using a heuristic:
 //   - if target is a vessel, target biggest unoccupied port
 //   - (else target is already a port)
-//   - find port on ship with same mass
+//   - find port on ship with rougly same mass
+// TODO: Invert the logic: Find a docking port on the ship and match it o a docking port on the target.
+// TODO: Use a better way to match port sizes. Mass isn't reliable at all. 
```

Comment:
It looks like kOS finally has a way to compare two ports for compatibility!
https://ksp-kos.github.io/KOS/structures/vessels/dockingport.html#attribute:DOCKINGPORT:NODETYPE

Reply:
Yes, I already have it working with this in my local copy!